docs/UserGuide.adoc

```diff
@@ -60,47 +60,47 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 Format: `help`
 
-=== Adding a person: `add`
```

Comment:
The User Guide must be changed completely. I will change it later, so this UG code is ok.